pkgs/by-name/go/got/package.nix

```diff
@@ -37,7 +41,15 @@ stdenv.mkDerivation (finalAttrs: {
     substituteInPlace configure --replace-fail 'xdarwin' 'xhomebrew'
   '';
 
-  env.NIX_CFLAGS_COMPILE = toString (lib.optionals stdenv.isDarwin [
+  env.NIX_CFLAGS_COMPILE = toString (
+  lib.optionals (defaultEditor != null) [
+    ''-DGOT_DEFAULT_EDITOR="${lib.getExe defaultEditor}"''
+  ] ++ lib.optionals withSsh [
+    ''-DGOT_DIAL_PATH_SSH="${lib.getExe openssh}"''
+    ''-DGOT_TAG_PATH_SSH_KEYGEN="${lib.getBin openssh}/bin/ssh-keygen"''
```

Comment:
this could use `getExe'` but that's splitting hairs I suppose

Reply:
Didn't know `getExe'` existed, thanks for bringing this to my attention, @eclairevoyant, much appreciated 🙂 

I'll update this PR accordingly.